collect_app/src/main/java/org/odk/collect/android/application/Collect.java

```diff
@@ -110,8 +110,8 @@
     AuthDialogUtility authDialogUtility;
     @Inject
     LocaleHelper localeHelper;
-
-    private ActivityLogger activityLogger;
+    @Inject
```

Comment:
I think a discussion was started about removing the `ActivityLogger` from the codebase. Do we want to do that soon since we are now investing time creating injections for it across the app?